cohort-engine-api/src/main/java/com/ibm/cohort/engine/api/service/FHIRRestUtils.java

```diff
@@ -175,6 +240,11 @@ public static IGenericClient getFHIRClient(String fhirEndpoint, String userName,
 		if(authString == null || authString.trim().isEmpty()) {
 			throw new IllegalArgumentException("No data in HTTP authorization header. FHIR server credentials must be passed using HTTP Basic Authentication");
 		}
+		
+		//If OAuth authentication is being used, parse the bearer token value from the headers and return it
+		if(authString.startsWith("Bearer")) {
```

Comment:
```suggestion
		if(authString.startsWith("Bearer ")) {
```

technically, this should be `Bearer<space>`
_(pushes glasses up nose)_

Reply:
I will accept your suggestion, but only because you wear glasses.